src/storage/query/GetPropProcessor.cpp

```diff
@@ -22,6 +22,9 @@ void GetPropProcessor::process(const cpp2::GetPropRequest& req) {
 
 void GetPropProcessor::doProcess(const cpp2::GetPropRequest& req) {
   spaceId_ = req.get_space_id();
+  // negative number means no limit
```

Comment:
The first letter of comments should be capitalized

Reply:
Done.